functions.php

```diff
@@ -129,12 +132,40 @@ function ampconf_widgets_init() {
 }
 add_action( 'widgets_init', 'ampconf_widgets_init' );
 
+/**
+ * Filters the AMP custom css to include the CSS from out theme.
+ *
+ * @param string $css The combined custom css.
+ *
+ * @return string $css
+ */
+function ampconf_custom_styles( $css ) {
+	$path   = get_template_directory() . '/assets/dist/css/main.css';
+	$styles = file_get_contents( $path ); // phpcs:ignore WordPress.WP.AlternativeFunctions -- Not a remote file.
+
+	return $styles . $css;
+}
+add_filter( 'amp_custom_styles', 'ampconf_custom_styles' );
+
+/**
+ * Adds custom component scripts to the document.
+ *
+ * @param array $amp_scripts AMP Component scripts, mapping component names to component source URLs.
+ *
+ * @return array
+ */
+function ampconf_amp_component_scripts( $amp_scripts ) {
+	$amp_scripts['amp-form'] = 'https://cdn.ampproject.org/v0/amp-form-0.1.js';
+	$amp_scripts['amp-bind'] = 'https://cdn.ampproject.org/v0/amp-bind-0.1.js';
+
+	return $amp_scripts;
+}
+add_filter( 'amp_component_scripts', 'ampconf_amp_component_scripts' );
+
 /**
  * Enqueue scripts and styles.
  */
 function ampconf_scripts() {
```

Comment:
We can get rid of this function since assets are dequeued by the AMP plugin. (We will add the necessary fallback for .org when we get to that story)

Reply:
This is actually not accurate. Assets don't get dequeued, but rather the printing is prevented. And with https://github.com/Automattic/amp-wp/pull/887 actually the enqueued assets are used for printing.